homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -80,11 +127,15 @@ def update(self):
         """
         try:
             self.client.connect()
```

Comment:
I'm a bit confused here. So this code will actually open and close the serial connection once PER entity. Why not keep it open while Home Assistant is running ?

Reply:
The solar inverter is powered by sunlight so every day it loses comms at least once.  It needs to retry periodically until sunrise in a way that doesn't generate errors.

I'm open to suggestions for better ways to do this!

Thanks for all the comments, will update in the next few days.

Reply:
Currently left as open/close which enables comms problems to be reset each day.